src/A1_csv_to_json.py

```diff
@@ -13,9 +13,9 @@
 For each storage there needs to be another file named exactly as the according
 storage-column in "energyStorage.csv", usually this is "storage_01",
 "storage_02" etc. Please stick to this convention.
-After all the function "infer_resources()" reads all csv that are stored in
-the folder "/mvs_eland/inputs/elements/csv"/the input directory and creates one
-json input file for mvs
+After all the function "infer_resources()" reads all csv files that are stored
+in the folder "/mvs_eland/inputs/elements/csv"/the input directory and creates
```

Comment:
```suggestion
in the folder provided by the user (input_directory) and creates
```
The function `infer_resources` is nowhere to be found, maybe the whole text needs to be reworked?